sunpy/data/test/sept/sept_ahead_ele_asun_2015_001_1d_l2_v03.dat.txt

```diff
@@ -0,0 +1,56 @@
+# - Energy windows (MeV): 0.0100;0.0100;0.0100;0.0100;0.0200;0.0200;0.0200;0.0200;0.0300;0.0300;0.0300;0.0400;0.0400;0.0400;0.0500;
+#
+#Time resolution: 1 day
+#Time stamps mark center of averaging interval (rounded to the second)
+#Energy bins 02 and 03 should not be used for quantitative studies. 
+#Electron intensities are not corrected for ion contamination 
+#Bad/missing data are filled with -9999.9 (float) or -9999 (integer)
+#Intensities and uncertainties are given in 1/(cm^2 s sr MeV)
+#
+#Column 1 : Julian date (double-precision float). Julian date 0.0 is Jan. 1, 4713 B.C. at 12:00:00
+#Column 2 : Year (integer)
+#Column 3 : Fractional day of year (double-precision float)
+#Column 4 : Hour of day (integer)
+#Column 5 : Minute of hour (integer)
+#Column 6 : Second of minute (integer)
+#Column 7 : Bin 02 (  45.0-  55.0 keV) electron intensity (float)
+#Column 8 : Bin 03 (  55.0-  65.0 keV) electron intensity (float)
+#Column 9 : Bin 04 (  65.0-  75.0 keV) electron intensity (float)
+#Column 10: Bin 05 (  75.0-  85.0 keV) electron intensity (float)
+#Column 11: Bin 06 (  85.0- 105.0 keV) electron intensity (float)
+#Column 12: Bin 07 ( 105.0- 125.0 keV) electron intensity (float)
+#Column 13: Bin 08 ( 125.0- 145.0 keV) electron intensity (float)
+#Column 14: Bin 09 ( 145.0- 165.0 keV) electron intensity (float)
+#Column 15: Bin 10 ( 165.0- 195.0 keV) electron intensity (float)
+#Column 16: Bin 11 ( 195.0- 225.0 keV) electron intensity (float)
+#Column 17: Bin 12 ( 225.0- 255.0 keV) electron intensity (float)
+#Column 18: Bin 13 ( 255.0- 295.0 keV) electron intensity (float)
+#Column 19: Bin 14 ( 295.0- 335.0 keV) electron intensity (float)
+#Column 20: Bin 15 ( 335.0- 375.0 keV) electron intensity (float)
+#Column 21: Bin 16 ( 375.0- 425.0 keV) electron intensity (float)
+#Column 22: Bin 02 Uncertainty (float)
+#Column 23: Bin 03 Uncertainty (float)
+#Column 24: Bin 04 Uncertainty (float)
+#Column 25: Bin 05 Uncertainty (float)
+#Column 26: Bin 06 Uncertainty (float)
+#Column 27: Bin 07 Uncertainty (float)
+#Column 28: Bin 08 Uncertainty (float)
+#Column 29: Bin 09 Uncertainty (float)
+#Column 30: Bin 10 Uncertainty (float)
+#Column 31: Bin 11 Uncertainty (float)
+#Column 32: Bin 12 Uncertainty (float)
+#Column 33: Bin 13 Uncertainty (float)
+#Column 34: Bin 14 Uncertainty (float)
+#Column 35: Bin 15 Uncertainty (float)
+#Column 36: Bin 16 Uncertainty (float)
+#Column 37: Accumulation time in seconds (float) 
+#BEGIN DATA
+2457024.000000 2015   1.500000 12 00 00   3.6066E+03   2.2359E+03   1.5261E+03   1.1767E+03   8.1142E+02   5.3320E+02   3.1059E+02   2.1448E+02   1.4719E+02   8.9111E+01   6.0667E+01   3.2879E+01   2.1808E+01   1.4287E+01   1.0683E+01   1.8600E+01   1.1015E+01   8.9576E+00   7.8657E+00   4.5083E+00   3.6205E+00   2.7014E+00   2.2857E+00   1.5390E+00   1.1763E+00   9.7918E-01   6.2150E-01   5.5204E-01   5.0627E-01   4.5838E-01   1.8832E+04
```

Comment:
1 day files have only one line data in them so basically thats the entire original file, look for yourself: http://www2.physik.uni-kiel.de/stereo/data/sept/level2/ahead/1d/2015/